uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-assignment_test.js

```diff
@@ -0,0 +1,104 @@
+define([
+    'common/angular',
+    'mocks/assignment',
+    'common/angularMocks',
+    'tasks-assignments/app'
+], function (angular, Mock) {
+    'use strict';
+
+    describe('ModalAssignmentCtrl', function () {
+        var ctrl, modalInstance, scope, AssignmentService, TaskService, DocumentService, ContactService, HR_settings;
+
+        beforeEach(module('civitasks.appDashboard'));
+        beforeEach(inject(function ($controller, $rootScope) {
+            modalInstance = {
+                close: jasmine.createSpy('modalInstance.close'),
+                dismiss: jasmine.createSpy('modalInstance.dismiss'),
+                result: {
+                    then: jasmine.createSpy('modalInstance.result.then')
+                }
+            };
+
+            scope = $rootScope.$new();
+
+            AssignmentService = {};
+            TaskService = {};
+            DocumentService = {};
+            ContactService = {};
+            HR_settings = {
+                DATE_FORMAT: 'DD/MM/YYYY'
+            };
+
+            ctrl = $controller('ModalAssignmentCtrl', {
+                $scope: scope,
+                $modalInstance: modalInstance,
+                AssignmentService: AssignmentService,
+                TaskService: TaskService,
+                DocumentService: DocumentService,
+                ContactService: ContactService,
+                data: {},
+                config: {},
+                settings: {},
+                HR_settings: HR_settings
+            });
+        }));
+
+        it('Controller should be defined', function () {
+            expect(ctrl).toBeDefined()
```

Comment:
I'm not sure this is a meaningful test, if the `ModalAssignmentCtrl` would not exist, then the entire test suite would throw an error, i don't believe you will get into this test example anyway

If you want to test this, maybe you could rewrite it as something like this:
```js
expect(initController).not.toThrow();

function initController() {
    ctrl = $controller('ModalAssignmentCtrl', { /* .. */ });
}
```